pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -301,15 +296,13 @@ WITH all_submissions_with_files AS (
         row_number() OVER (PARTITION BY v.id ORDER BY s.date) AS submission_number,
         (row_number() OVER (PARTITION BY v.id ORDER BY s.date DESC, s.id DESC)) = 1 AS final_submission_per_variant,
         (row_number() OVER (PARTITION BY v.id ORDER BY s.score DESC NULLS LAST, s.date DESC, s.id DESC)) = 1 AS best_submission_per_variant,
-        (row_number() OVER (PARTITION BY g.id, v.id, s.id)) = 1 AS unique_group,
         g.name AS group_name
     FROM
         assessments AS a
         JOIN assessment_instances AS ai ON (ai.assessment_id = a.id)
         LEFT JOIN group_configs AS gc ON (gc.assessment_id = a.id)
         LEFT JOIN groups AS g ON (g.id = ai.group_id AND g.group_config_id = gc.id)
-        LEFT JOIN group_users AS gu ON (gu.group_id = g.id)
-        JOIN users AS u ON (u.user_id = ai.user_id OR u.user_id = gu.user_id)
+        JOIN users AS u ON (u.user_id = ai.user_id)
```

Comment:
Should this be a LEFT JOIN? If it's just a plain JOIN it will require that `ai.user_id`) is NOT NULL, which will exclude group work assessment instances.

Reply:
My bad, this one was missing. Fixed now.